mico-admin/src/app/api/semantic-version.ts

```diff
@@ -43,3 +43,46 @@ export function versionComparator(versionA: string, versionB: string): number {
     }
     return 0;
 }
+
+export function incrementVersion(version: String) {
```

Comment:
add parameter to choose which version (major/minor/patch) to increment. (Possibly with default of minor).
Also add doc comment as this is a valuable utility function.